fe/fe-core/src/test/java/com/starrocks/lake/StarMgrMetaSyncerTest.java

```diff
@@ -554,4 +554,76 @@ Future<DeleteTabletResponse> deleteTablet(DeleteTabletRequest request) {
         // can delete the shards, because the error is INVALID_ARGUMENT
         Assert.assertEquals(0, allShardIds.size());
     }
+
+    @Test
+    public void testForceDelete() {
+        Config.meta_sync_force_delete_shard_meta = true;
+        Config.shard_group_clean_threshold_sec = 0;
+        long groupIdToClear = shardGroupId + 1;
+        List<Long> allShardGroupId = Lists.newArrayList(groupIdToClear);
+        // build shardGroupInfos
+        List<Long> allShardIds = Stream.of(1000L, 1001L, 1002L, 1003L).collect(Collectors.toList());
+        int numOfShards = allShardIds.size();
```

Comment:
this variable is not used. this is the reason the sonar check fails.

Reply:
fixed